cellprofiler/modules/shrinkobjectstopoints.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+
+"""
+ShrinkObjectsToPoints
+======================
+
+**ShrinkObjectsToPoints** will transform a set of objects into a label image with single points
+representing each object. The location of each point corresponds to the centroid of the input objects.
+"""
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.setting
+import numpy
+import skimage.measure
+
+
+class ShrinkObjectsToPoints(cellprofiler.module.ObjectProcessing):
+
+    module_name = "ShrinkObjectsToPoints"
+
+    category = "Advanced"
+
+    variable_revision_number = 1
+
+    def create_settings(self):
+
+        super(ShrinkObjectsToPoints, self).create_settings()
+
+        self.x_name.text = "Input objects"
```

Comment:
This is already defined as

```python
self.x_name = cellprofiler.setting.ObjectNameSubscriber(
    "Select the input object",
    doc="Select the object you want to use."
)
```

Perhaps only `self.x_name.doc` needs updating?